tests/snmp/test_snmp_cpu.py

```diff
@@ -1,6 +1,12 @@
 import pytest
 import time
 from ansible_host import AnsibleHost
+from common.utilities import wait_until
+
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
too many duplicated code, suggest to move to one place.

Reply:
Are there any examples of use one setup function on different pytest file? I add one function check_snmp_ready() on another python file (e.g. test_snmp.py), and add function setup_check_snmp_ready() on every test_snmp_xxx.py to call check_snmp_ready() to wait for snmp service to be started. But I'm not sure if that's correct.